hdc/algo/_version.py

```diff
@@ -1,2 +1,2 @@
 """Version only in this file."""
-__version__ = "0.2.0"
+__version__ = "0.2.1"
```

Comment:
you consider this a patch release? We've got `0.3.0` pending with the other PR. Should we do `0.2.1` here and bump with the other one?

Reply:
anything goes before 1.0 according semantic versioning standard. I just needed a version bump to test conda install. I don't think our user-base is wide enough to think too much about these things.

Reply:
fair enough